conda_forge_tick/cli.py

```diff
@@ -151,11 +151,20 @@ def update_upstream_versions(
 
 
 @main.command(name="auto-tick")
+@click.argument(
+    "package",
+    required=False,
```

Comment:
```suggestion
    required=False,
    default=None,
    type=str,
```

This will help things be a bit more explicit on what happens if nothing is supplied and what type is expected. 

We also need to add a help string.

Finally, this name should be changed from `package` to `feedstock` and it should specify if the argument is `blah-feedstock` or just `blah`.

Reply:
I applied your suggestion consistently across all CLI commands and method signatures.
The only help string we have for arguments is the docstring below, I made sure it is explained there.